chain/multisig.go

```diff
@@ -8,35 +8,35 @@ import (
 	"github.com/filecoin-project/chain-validation/chain/types"
 )
 
```

Comment:
These files are generated, if this is a change we want then I propose we regenerate/change them all for consistency (I  volunteer to make that change if so). Is this change necessary? Seems like its more about style.

Reply:
Oh, I didn't realise it was generated. Ok I'll revert, but yes could you please
- change the codegen so as not to copy params objects, but pass a pointer
- include a comment saying this is generated code and how to re-gen it